src/Icons/src/DependencyInjection/UXIconsExtension.php

```diff
@@ -64,17 +66,23 @@ public function getConfigTreeBuilder(): TreeBuilder
         return $builder;
     }
 
+    /**
+     * @param array<string, mixed> $config
+     */
     public function getConfiguration(array $config, ContainerBuilder $container): ConfigurationInterface
     {
         return $this;
     }
 
-    protected function loadInternal(array $mergedConfig, ContainerBuilder $container): void // @phpstan-ignore-line
+    /**
+     * @param array<string, mixed> $mergedConfig
+     */
+    protected function loadInternal(array $mergedConfig, ContainerBuilder $container): void
     {
         $loader = new PhpFileLoader($container, new FileLocator(__DIR__.'/../../config'));
         $loader->load('services.php');
 
-        if (isset($container->getParameter('kernel.bundles')['TwigComponentBundle'])) {
+        if (is_array($kernelBundles = $container->getParameter('kernel.bundles')) && isset($kernelBundles['TwigComponentBundle'])) {
```

Comment:
Can't we use ?? instead? This will saves us 2 function calls:
```suggestion
        if ($container->getParameter('kernel.bundles')['TwigComponentBundle'] ?? null) {
```

Reply:
85     Cannot access offset 'TwigComponentBundle' on array|bool|float|int|string|UnitEnum.